tests/lib/rules/no-title-property-in-meta.test.ts

```diff
@@ -25,15 +25,15 @@ ruleTester.run('no-title-property-in-meta', rule, {
 
   invalid: [
     {
-      code: "export default { component: Button, title: 'Button' }",
+      code: "export default { title: 'Button', component: Button }",
       errors: [
         {
           messageId: 'noTitleInMeta',
-          type: AST_NODE_TYPES.ExportDefaultDeclaration,
+          type: AST_NODE_TYPES.Property,
           suggestions: [
             {
               messageId: 'removeTitleInMeta',
-              output: 'export default { component: Button,  }',
+              output: 'export default {  component: Button }',
```

Comment:
what's up with the extra space?

Reply:
it's related to what you mentioned before:

> Ideally we'd be modifying the DOM and then generating using prettier or something, but I guess we want to preserve the user's formatting as much as possible...

Not sure what's the best way to improve this without running prettier